google/resource_sql_database_instance_test.go

```diff
@@ -521,6 +521,33 @@ func TestAccSqlDatabaseInstance_multipleOperations(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_basic_with_user_labels(t *testing.T) {
+	t.Parallel()
+
+	var instance sqladmin.DatabaseInstance
+	databaseID := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccSqlDatabaseInstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: fmt.Sprintf(
```

Comment:
Can you add a new step in this test to test the update path for labels?

Reply:
done